presto-main/src/main/java/com/facebook/presto/testing/MaterializedRow.java

```diff
@@ -52,10 +52,11 @@ public MaterializedRow(int precision, List<Object> values)
 
     private static Object processValue(int precision, Object value)
     {
+        checkArgument(!(value instanceof Integer || value instanceof Short || value instanceof Byte), "INTEGER, SMALLINT and TINYINT are not yet supported");
         if (value instanceof Double || value instanceof Float) {
             return new ApproximateDouble(((Number) value).doubleValue(), precision);
         }
-        if (value instanceof Number) {
+        if (value instanceof Long) {
             return ((Number) value).longValue();
```

Comment:
IntelliJ should warn you about this.  If it is not, ask @electrum to help you get it configured correctly.